db/schema.rb

```diff
@@ -35,7 +35,7 @@
   add_index "builds", ["project_id"], name: "index_builds_on_project_id", using: :btree
 
   create_table "commands", force: :cascade do |t|
-    t.text     "command",    limit: 10485760
+    t.text     "command",    limit: 16777215
```

Comment:
I'm not sure what the previous value was, but this is 2^24 which is what a MySQL MEDIUMTEXT column gets set to.